Presentation.Web/app/components/data-processing/data-processing-registration-overview.controller.ts

```diff
@@ -146,7 +146,16 @@
                             .withTitle("Databehandleraftale er indgået")
                             .withId("agreementConcluded")
                             .withStandardWidth(100)
-                            .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.Contains)
+                            .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.FixedValueRange)
+                            .withFixedValueRange
```

Comment:
@JacobMalling FYI - nu har vi også filtrering med kendte udfaldsrum så man ikke skal skrive fritekst for den slags kolonner